### PR TITLE
BGDIDIC-2837: convert float to int

### PR DESCRIPTION
**Pull request by faselm**

### CODE REVIEW

**Review by ltclm — CHANGES_REQUESTED** [review]
please also do as proposed by @marionb 
```
int(round(c['attributes']['elevation'])) if c['attributes']['elevation'] is not None else '-'
```
seems to be more robust if the attribute contains NULL values

File: chsdi/templates/htmlpopup/vbs_ladestationen.mako [hunk 1]
```diff
@@ -46,11 +46,11 @@
     </tr>
     <tr>
         <td class="cell-left">${_('ch.vbs.ladestationen.XKoordinate')}</td>
-        <td>${c['attributes']['x_koordinate'] or '-'}</td>
+        <td>${int(round(c['attributes']['x_koordinate'])) if c['attributes']['x_koordinate'] is not None else '-'}</td>
```
Comment (marionb) [inline chsdi/templates/htmlpopup/vbs_ladestationen.mako:49]: Don't these coordinates already come as int values? When I look at the the layer in dev they are always `xyz.0`
In this case I think it makes more sens to integrate them in t the DB as int so you don't have to round them here and use the actual values the data owner provides?

**Review by rebert — APPROVED** [review]

File: chsdi/templates/htmlpopup/vbs_ladestationen.mako (hunk 1 above)
Comment (marionb) [reply chsdi/templates/htmlpopup/vbs_ladestationen.mako:49]: Yes it is in the table that need to be changed as well as the model here:
- https://github.com/geoadmin/mf-chsdi3/blob/develop-2024-07-03/chsdi/models/vector/vbs.py#L409
- https://github.com/geoadmin/mf-chsdi3/blob/develop-2024-07-03/chsdi/models/vector/vbs.py#L417-L418

If you add these changes  you will not need this extra code and the make will be easier to read and maintain.

**Review by marionb — CHANGES_REQUESTED** [review]
see my comment above

File: chsdi/templates/htmlpopup/vbs_ladestationen.mako (hunk 1 above)
Comment (ltclm) [reply chsdi/templates/htmlpopup/vbs_ladestationen.mako:49]: yes, the values are integer in the identify result but the htmlpopup converts them for unknown reasons into x.0 numbers...

**Review by ltclm — APPROVED** [review]

**Review by marionb — APPROVED** [review]

**Pull request merged**